src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const listPlace = document.querySelector('ul');
+const list = document.querySelectorAll('li');
+
+function sortList(item) {
+  const salaryList = [...item].sort((a, b) =>
+    +b.dataset.salary.slice(1).replace(',', '')
+ - +a.dataset.salary.slice(1).split(',').join(''));
```

Comment:
I think it is better to create additional function that converts salary into needed format